src/helper/validator.js

```diff
@@ -0,0 +1,86 @@
+class validator {
+    static userValidator(user, userList) {
+      if (
+        user.hasOwnProperty("email") &&
```

Comment:
Please explore some open-source options to do basic data type checks, required field checks, and even the complicated validations.